README.md

```diff
@@ -29,7 +29,7 @@
   <img src="docs/env_anim/cvrp.gif" alt="CVRP" width="30%" />
   <img src="docs/env_anim/rubiks_cube.gif" alt="RubiksCube" width="30%" />
   <img src="docs/env_anim/game_2048.gif" alt="Game2048" width="30%" />
-  <img src="docs/env_anim/minesweeper.gif" alt="Minesweeper" width="30%" />
+  <img src="docs/env_anim/sudoku.gif" alt="Sudoku" width="30%" />
```

Comment:
Why is minesweeper getting deleted?

Reply:
We are only keeping 9 GIF (3x3 grid) on the main readme. Minesweeper was voted as less cool than Sudoku. Are you okay with this?

Reply:
Bit sad but I'll live

Reply:
> Here is the corresponding training curve. ![image](https://user-images.githubusercontent.com/48136643/237801874-c39dae24-e39c-4ccb-893e-56f8b8b18d59.png)

So 50% of boards are solved just by taking random (valid) actions?

Reply:
Yeah some of the boards of the `very-easy` datasets only have a few empty cells.